src/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx

```diff
@@ -153,13 +153,17 @@ export const FeaturedCollectionsRailsContainer = createFragmentContainer(
   }
 )
 
-const Container = styled(Box)`
+const FeaturedCollectionsContainer = styled(Box)`
+  border-top: 1px solid ${color("black10")};
+`
+
+const Container = styled(Box)<{ width: string }>`
   border: 1px solid ${color("black10")};
-  width: 365px;
+  width: ${({ width }) => width};
```

Comment:
`Box` should [already contain width](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Box/Box.tsx#L73)

Reply:
Thanks for pointing this out!